src/server/server_dispatch.c

```
@@ -2253,13 +2253,17 @@ void bc_handle_packet(const bc_addr_t *from, u8 *data, int len)
          * Fragment messages need a 5-byte ACK with frag_idx so the
          * client drains the correct retransmit queue entry. */
         if (tmsg->type == BC_TRANSPORT_RELIABLE && (tmsg->flags & 0x80)) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)